### PR TITLE
en: add crd-based backup and restore docs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

This PR is migrated from https://github.com/pingcap/docs/pull/1945:

- Add CRD-based backup and restoration documents for TiDB Operator v1.1.
- Change the title of the original backup-and-restore document.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-1.1** and **needs-cherry-pick-1.0**.

### What is the related PR or file link(s)?

- This PR is translated from:
    - https://github.com/pingcap/docs-cn/pull/2055
    - https://github.com/pingcap/docs-cn/pull/2236
    - https://github.com/pingcap/docs-cn/pull/2379
- Other reference link(s):

### CODE REVIEW

File: en/TOC.md [hunk 1]
```diff
@@ -29,7 +29,13 @@
...
   - [Restart a TiDB Cluster](restart-a-tidb-cluster.md)
   - [Maintain a Hosting Kubernetes Node](maintain-a-kubernetes-node.md)
-  - [Backup and Restore](backup-and-restore.md)
+  + Backup and Restore
+    - [Helm Charts-Based Backup and Restoration](backup-and-restore-using-helm-charts.md)
+    + CRD-Based Backup and Restoration
+      - [Back up Data to GCS](backup-to-gcs.md)
+      - [Restore Data From GCS](restore-from-gcs.md)
+      - [Back up Data to S3-Compatible Storage](backup-to-s3.md)
+      - [Restore Data From S3-Compatible Storage](restore-from-s3.md)
```
Comment (kennytm) [inline en/TOC.md]: is there any reason why "to" and "up" are in lower-case but "From" is capitalized?

Comment (TomShawn) [reply en/TOC.md]: Thanks for pointing it out. According to the [APA style guide](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) on title case capitalization, words of four letters or more should be capitalized.

Comment (kennytm) [reply en/TOC.md]: Okay 🙂

Comment (TomShawn) [reply en/TOC.md]: Sorry, I've confirmed with my team that our documentation adopts the Chicago Style, which means that `from` should not be capitalized.

File: en/backup-to-gcs.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+After creating the `Backup` CR, execute the following command to check the backup status:
+
+{{< copyable "shell-regular" >}}
+
+ ```shell
+ kubectl get bk -n test1 -owide
+ ```
+
+More `Backup` CRs are as described follows:
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
More `Backup` CRs are described as follows:
```

File: en/backup-to-gcs.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+* `.spec.metadata.namespace`: the namespace where the `Backup` CR is located.
+* `.spec.from.host`: the address of the TiDB cluster to be backed up.
+* `.spec.from.port`: the port of the TiDB cluster to be backed up.
+* `.spec.from.user`: the accessing user of the TiDB cluster to be backed up.
+* `.spec.from.tidbSecretName`: the secrete of the credential needed by the TiDB cluster to be backed up.
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
* `.spec.from.tidbSecretName`: the secret of the credential needed by the TiDB cluster to be backed up.
```

File: en/backup-to-gcs.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+* `.spec.storageClassName`: the persistent volume (PV) type specified for the backup operation. If this item is not specified, the value of the `default-backup-storage-class-name` parameter (`standard` by default, specified when TiDB Operator is started) is used by default.
```
Comment (ran-huang) [inline en/backup-to-gcs.md]: How about this?
```suggestion
* `.spec.storageClassName`: the persistent volume (PV) type specified for the backup operation. If this item is not specified, the value of the `default-backup-storage-class-name` parameter is used by default. This parameter is specified when TiDB Operator is started, and is set to `standard` by default.
```

File: en/backup-to-gcs.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than size of the TiDB cluster to be backed up.
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than the size of the TiDB cluster to be backed up.
```

File: en/backup-to-gcs.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+* `authenticatedRead`
+* `private`
+* `projectPrivate`
+* `publicRead`
+* `publicReadWrite`
+
+If the bucket ACL policy is not configured, the `private` policy is used by default. For the detailed description of these access control policies, refer to [GCS documentation](https://cloud.google.com/storage/docs/access-control/lists).
+
+After creating the `Backup` CR, execute the following command to check the backup status:
```
Comment (ran-huang) [inline en/backup-to-gcs.md]: Keep the wording consistent with line 183.
```suggestion
After creating the `Backup` CR, use the following command to check the backup status:
```

File: en/backup-to-gcs.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+After creating the scheduled full backup, use the following command to check the backup status:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get bks -n test1 -owide
+```
+
+Execute the following command to check all the backup items:
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
Use the following command to check all the backup items:
```

File: en/backup-to-gcs.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+
+{{< copyable "shell-regular" >}}
+
+ ```shell
+ kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-gcs -n test1
+ ```
+
+From the above example, you can see that the `backupSchedule` configuration consists of two part. One is the unique configuration of `backupSchedule` and the other is `backupTemplate`. `backupTemple` specifies the configuration related to the GCS storage, which is the same with the configuration of the ad-hoc full backup to GCS (refer to [GCS backup process](#ad-hoc-backup-process) for details). The following are the unique configuration items of `backupSchedule`:
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
From the above example, you can see that the `backupSchedule` configuration consists of two parts. One is the unique configuration of `backupSchedule`, and the other is `backupTemplate`. `backupTemple` specifies the configuration related to the GCS storage, which is the same as the configuration of the ad-hoc full backup to GCS (refer to [GCS backup process](#ad-hoc-backup-process) for details). The following are the unique configuration items of `backupSchedule`:
```

File: en/backup-to-gcs.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+
++ `.spec.maxBackups`: A backup retention policy, which determines the maximum number of backup items to be retained. When this value is exceeded, the outdated backup items will be deleted. If you set this configuration item to `0`, all backup items are retained.
++ `.spec.maxReservedTime`: A backup retention policy based on time. For example, if you set the value of this configuration to `24h`, backup items only of recent 24 hours are retained. All backup items out of this time are deleted. For the time format, refer to [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration). If you have set the maximum number of backup items and the longest retention time of backup items at the same time, the latter setting takes effect.
```
Comment (ran-huang) [inline en/backup-to-gcs.md]:
```suggestion
+ `.spec.maxReservedTime`: A backup retention policy based on time. For example, if you set the value of this configuration to `24h`, only backup items within the recent 24 hours are retained. All backup items out of this time are deleted. For the time format, refer to [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration). If you have set the maximum number of backup items and the longest retention time of backup items at the same time, the latter setting takes effect.
```

File: en/backup-to-s3.md [hunk 10]
```diff
@@ -0,0 +1,271 @@
...
+
+After creating the `Backup` CR, use the following command to check the backup status:
+
+{{< copyable "shell-regular" >}}
+
+ ```shell
+ kubectl get bk -n test1 -owide
+ ```
+
+More `Backup` CRs are as described follows:
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
More `Backup` CRs are described as follows:
```

File: en/backup-to-s3.md [hunk 11]
```diff
@@ -0,0 +1,271 @@
...
+
+* `.spec.metadata.namespace`: the namespace where the `Backup` CR is located.
+* `.spec.from.host`: the address of the TiDB cluster to be backed up.
+* `.spec.from.port`: the port of the TiDB cluster to be backed up.
+* `.spec.from.user`: the accessing user of the TiDB cluster to be backed up.
+* `.spec.from.tidbSecretName`: the secrete of the credential needed by the TiDB cluster to be backed up.
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
* `.spec.from.tidbSecretName`: the secret of the credential needed by the TiDB cluster to be backed up.
```

File: en/backup-to-s3.md [hunk 12]
```diff
@@ -0,0 +1,271 @@
...
+* `.spec.storageClassName`: the persistent volume (PV) type specified for the backup operation. If this item is not specified, the value of the `default-backup-storage-class-name` parameter (`standard` by default, specified when TiDB Operator is started) is used by default.
+* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than size of the TiDB cluster to be backed up.
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than the size of the TiDB cluster to be backed up.
```

File: en/backup-to-s3.md [hunk 13]
```diff
@@ -0,0 +1,271 @@
...
+* `wasabi`: Wasabi Object Storage
+* `other`: Any other S3 compatible provider
+
+## Scheduled full backup to S3-compatible storage
+
+You can set a backup policy to perform scheduled backups of the TiDB cluster, and set a backup retention policy to avoid excessive backup items. A scheduled full backup is described by a custom `BackupSchedule` CR object. A full backup is triggered at each backup time point. Its underlying implementation is the ad-hoc full backup.
+
+### Prerequisites for scheduled backup
+
+The prerequisites for the scheduled backup is the same with the [prerequisites for ad-hoc backup](#prerequisites-for-ad-hoc-backup).
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
The prerequisites for the scheduled backup is the same as the [prerequisites for ad-hoc backup](#prerequisites-for-ad-hoc-backup).
```

File: en/backup-to-s3.md [hunk 14]
```diff
@@ -0,0 +1,271 @@
...
+          secretName: s3-secret
+          # region: us-east-1
+          # storageClass: STANDARD_IA
+          # acl: private
+          # endpoint:
+        storageClassName: local-storage
+        storageSize: 10Gi
+    ```
+
++ Create the `BackupSchedule` CR to enable the scheduled full backup to Amazon S3:
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
+ Create the `BackupSchedule` CR to enable the scheduled full backup to Ceph:
```

File: en/backup-to-s3.md [hunk 15]
```diff
@@ -0,0 +1,271 @@
...
+
+After creating the scheduled full backup, use the following command to check the backup status:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get bks -n test1 -owide
+```
+
+Execute the following command to check all the backup items:
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
Use the following command to check all the backup items:
```

File: en/backup-to-s3.md [hunk 16]
```diff
@@ -0,0 +1,271 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get bk -l tidb.pingcap.com/backup-schedule=demo1-backup-schedule-s3 -n test1
+```
+
+From the above two examples, you can see that the `backupSchedule` configuration consists of two part. One is the unique configurations of `backupSchedule` and the other is `backupTemplate`. `backupTemple` specifies the configuration related to the S3-compatible storage, which is the same with the configuration of the ad-hoc full backup to the S3-compatible storage (refer to [Ad-hoc backup process](#ad-hoc-backup-process) for details). The following are the unique configuration items of `backupSchedule`:
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
From the above two examples, you can see that the `backupSchedule` configuration consists of two parts. One is the unique configuration of `backupSchedule`, and the other is `backupTemplate`. `backupTemple` specifies the configuration related to the S3-compatible storage, which is the same as the configuration of the ad-hoc full backup to the S3-compatible storage (refer to [Ad-hoc backup process](#ad-hoc-backup-process) for details). The following are the unique configuration items of `backupSchedule`:
```

File: en/backup-to-s3.md [hunk 17]
```diff
@@ -0,0 +1,271 @@
...
+
++ `.spec.maxBackups`: A backup retention policy, which determines the maximum number of backup items to be retained. When this value is exceeded, the outdated backup items will be deleted. If you set this configuration item to `0`, all backup items are retained.
++ `.spec.maxReservedTime`: A backup retention policy based on time. For example, if you set the value of this configuration to `24h`, backup items only of recent 24 hours are retained. All backup items out of this time are deleted. For the time format, refer to [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration). If you have set the maximum number of backup items and the longest retention time of backup items at the same time, the latter setting takes effect.
```
Comment (ran-huang) [inline en/backup-to-s3.md]:
```suggestion
+ `.spec.maxReservedTime`: A backup retention policy based on time. For example, if you set the value of this configuration to `24h`, only backup items within the recent 24 hours are retained. All backup items out of this time are deleted. For the time format, refer to [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration). If you have set the maximum number of backup items and the longest retention time of backup items at the same time, the latter setting takes effect.
```

File: en/restore-from-gcs.md [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+     ```shell
+     kubectl get rt -n test2 -owide
+     ```
+
+In the above example, the backup data stored in the specified `spec.gcs.path` path on GCS is restored to the `spec.to.host` TiDB cluster. For the configuration of GCS, refer to [backup-gcs.yaml](backup-to-gcs.md#ad-hoc-backup-process).
+
+More `Restore` CRs are as described follows:
```
Comment (ran-huang) [inline en/restore-from-gcs.md]:
```suggestion
More `Restore` CRs are described as follows:
```

File: en/restore-from-gcs.md [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+
+* `.spec.metadata.namespace`: the namespace where the `Restore` CR is located.
+* `.spec.to.host`: the address of the TiDB cluster to be restored.
+* `.spec.to.port`: the port of the TiDB cluster to be restored.
+* `.spec.to.user`: the accessing user of the TiDB cluster to be restored.
+* `.spec.to.tidbSecretName`: the secrete of the credential needed by the TiDB cluster to be restored.
```
Comment (ran-huang) [inline en/restore-from-gcs.md]:
```suggestion
* `.spec.to.tidbSecretName`: the secret of the credential needed by the TiDB cluster to be restored.
```

File: en/restore-from-gcs.md [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+* `.spec.storageClassName`: the persistent volume (PV) type specified for the restoration. If this item is not specified, the value of the `default-backup-storage-class-name` parameter (`standard` by default, specified when TiDB Operator is started) is used by default.
+* `.spec.storageSize`: the PV size specified for the restoration. This value must be greater than size of the backed up TiDB cluster.
```
Comment (ran-huang) [inline en/restore-from-gcs.md]:
```suggestion
* `.spec.storageSize`: the PV size specified for the restoration. This value must be greater than the size of the backed up TiDB cluster.
```

File: en/restore-from-s3.md [hunk 21]
```diff
@@ -0,0 +1,85 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+     ```shell
+     kubectl get rt -n test2 -owide
+     ```
+
+In the above example, the backup data stored in the `spec.s3.path` path on the S3-compatible storage is restored to the `spec.to.host` TiDB cluster. For the configuration of the S3-compatible storage, refer to [backup-s3.yaml](backup-to-s3.md#ad-hoc-backup-process).
+
+More `Restore` CRs are as described follows:
```
Comment (ran-huang) [inline en/restore-from-s3.md]:
```suggestion
More `Restore` CRs are described as follows:
```

File: en/restore-from-s3.md [hunk 22]
```diff
@@ -0,0 +1,85 @@
...
+
+* `.spec.metadata.namespace`: the namespace where the `Restore` CR is located.
+* `.spec.to.host`: the address of the TiDB cluster to be restored.
+* `.spec.to.port`: the port of the TiDB cluster to be restored.
+* `.spec.to.user`: the accessing user of the TiDB cluster to be restored.
+* `.spec.to.tidbSecretName`: the secrete of the credential needed by the TiDB cluster to be restored.
```
Comment (ran-huang) [inline en/restore-from-s3.md]:
```suggestion
* `.spec.to.tidbSecretName`: the secret of the credential needed by the TiDB cluster to be restored.
```

File: en/restore-from-s3.md [hunk 23]
```diff
@@ -0,0 +1,85 @@
...
+* `.spec.storageClassName`: the persistent volume (PV) type specified for the restoration. If this item is not specified, the value of the `default-backup-storage-class-name` parameter (`standard` by default, specified when TiDB Operator is started) is used by default.
+* `.spec.storageSize`: the PV size specified for the restoration. This value must be greater than size of the backed up TiDB cluster.
```
Comment (ran-huang) [inline en/restore-from-s3.md]:
```suggestion
* `.spec.storageSize`: the PV size specified for the restoration. This value must be greater than the size of the backed up TiDB cluster.
```

File: zh/backup-and-restore-using-helm-charts.md [hunk 24]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 基于 Helm Charts 实现的 TiDB 集群备份恢复
 category: how-to
-aliases: ['/docs-cn/devmaintain/backup-and-store/']
+aliases: ['/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-store/','/docs-cn/stable/tidb-in-kubernetes/maintain/backup-and-store/','/docs-cn/v3.1/tidb-in-kubernetes/maintain/backup-and-store/']
```
Comment (ran-huang) [inline zh/backup-and-restore-using-helm-charts.md]:
```suggestion
aliases: ['/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs-cn/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs-cn/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/']
```

File: en/backup-to-s3.md [hunk 25]
```diff
@@ -0,0 +1,271 @@
...
+
+The backup method described in this document is implemented based on CustomResourceDefinition (CRD) in TiDB Operator v1.1 or later versions. For the backup method implemented based on Helm Charts, refer to [Back up and Restore TiDB Cluster Data Based on Helm Charts](backup-and-restore-using-helm-charts.md).
+
+## Ad-hoc full backup
+
+Ad-hoc full backup describes the backup by creating a `Backup` custom resource (CR) object. TiDB Operator performs the specific backup operation based on this `Backup` object. If an error occurs during the backup process, TiDB Operator does not retry and you need to handle this error manually.
+
+For the current S3-compatible storage types, Ceph and Amazon S3 work normally as tested. Therefore, this document shows examples in which the data of the `demo1` TiDB cluster in the `test1` Kubernetes namespace is backed up to Ceph and Amazon S3 respectively.
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply en/backup-to-s3.md]: "Theoretically, other storage types that are compatible with S3 can work normally as well." is making an assumption. I don't think this is an accurate or reliable statement, so I chose not to translate this sentence.

**Comment by TomShawn** [comment]
@lilin90 @ran-huang Comments addressed, PTAL again, thanks!

File: en/backup-and-restore-using-helm-charts.md [hunk 26]
```diff
@@ -1,14 +1,22 @@
...
+
+* [Back up Data to GCS](backup-to-gcs.md)
+* [Restore Data from GCS](restore-from-gcs.md)
+* [Back up Data to S3-Compatible Storage](backup-to-s3.md)
+* [Restore Data from S3-Compatible Storage](restore-from-s3.md)
+
+TiDB in Kubernetes supports two kinds of backup strategies (based on Helm charts):
 
 * [Full backup](#full-backup) (scheduled or ad-hoc): use [`mydumper`](https://pingcap.com/docs/stable/reference/tools/mydumper) to take a logical backup of the TiDB cluster.
 * [Incremental backup](#incremental-backup): use [`TiDB Binlog`](https://pingcap.com/docs/stable/reference/tidb-binlog/overview) to replicate data in the TiDB cluster to another database or take a real-time backup of the data.
```
Comment (lilin90) [inline en/backup-and-restore-using-helm-charts.md]:
```suggestion
* [Incremental backup](#incremental-backup): use [TiDB Binlog](https://pingcap.com/docs/stable/reference/tidb-binlog/overview) to replicate data from the TiDB cluster to another database or execute a real-time backup of the data.
```

File: en/backup-to-gcs.md [hunk 27]
```diff
@@ -0,0 +1,204 @@
+---
+title: Back up TiDB Cluster Data to GCS
+summary: Learn how to back up the TiDB cluster to GCS.
+category: how-to
+---
+
+# Back up TiDB Cluster Data to GCS
+
+This document describes how to back up the data of the TiDB cluster in Kubernetes to [Google Cloud Storage (GCS)](https://cloud.google.com/storage/docs/). "Backup" in this document refers to full backup (ad-hoc full backup and scheduled full backup). For the underlying implementation, [`mydumper`](https://pingcap.com/docs/stable/reference/tools/mydumper) is used to get the logic backup of the TiDB cluster, and then this backup data is sent to the remote GCS.
```
Comment (lilin90) [inline en/backup-to-gcs.md]: IMO, this phrase is not necessary.

```suggestion
This document describes how to back up the data of the TiDB cluster in Kubernetes to [Google Cloud Storage (GCS)](https://cloud.google.com/storage/docs/). "Backup" in this document refers to full backup (ad-hoc full backup and scheduled full backup). [`mydumper`](https://pingcap.com/docs/stable/reference/tools/mydumper) is used to get the logic backup of the TiDB cluster, and then this backup data is sent to the remote GCS.
```

File: en/backup-to-gcs.md [hunk 28]
```diff
@@ -0,0 +1,204 @@
...
+
+The backup method described in this document is implemented based on CustomResourceDefinition (CRD) in TiDB Operator v1.1 or later versions. For the backup method implemented based on Helm Charts, refer to [Back up and Restore TiDB Cluster Data Based on Helm Charts](backup-and-restore-using-helm-charts.md).
```
Comment (lilin90) [inline en/backup-to-gcs.md]:
```suggestion
The backup method described in this document is implemented using CustomResourceDefinition (CRD) in TiDB Operator v1.1 or later versions. For the backup method implemented using Helm Charts, refer to [Back up and Restore TiDB Cluster Data Using Helm Charts](backup-and-restore-using-helm-charts.md).
```

File: en/backup-to-gcs.md [hunk 29]
```diff
@@ -0,0 +1,204 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic backup-demo1-tidb-secret --from-literal=password=<password> --namespace=test1
+    ```
+
+### Ad-hoc backup process
+
+Create the `Backup` CR and back up data to GSC:
```
Comment (lilin90) [inline en/backup-to-gcs.md]: Suggesting using an ordered list for this section.

File: en/backup-to-gcs.md [hunk 30]
```diff
@@ -0,0 +1,204 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f backup-gcs.yaml
+```
+
+The `backup-gcs.yaml` file has the following content:
```
Comment (lilin90) [inline en/backup-to-gcs.md]: - Move this step to the 1st step.
- Update description.

```suggestion
1. In the `backup-gcs.yaml` file, edit `host`, `port`, `user`, `projectId` and save your changes.
```

File: en/backup-to-gcs.md [hunk 31]
```diff
@@ -0,0 +1,204 @@
...
+
+Create the `BackupSchedule` CR to enable the scheduled full backup to GCS:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl apply -f backup-schedule-gcs.yaml
+```
+
+The `backup-gcs.yaml` file has the following content:
```
Comment (lilin90) [inline en/backup-to-gcs.md]:
```suggestion
The `backup-schedule-gcs.yaml` file has the following content:
```

File: en/backup-to-gcs.md [hunk 32]
```diff
@@ -0,0 +1,204 @@
...
+
+You can set a backup policy to perform scheduled backups of the TiDB cluster, and set a backup retention policy to avoid excessive backup items. A scheduled full backup is described by a custom `BackupSchedule` CR object. A full backup is triggered at each backup time point. Its underlying implementation is the ad-hoc full backup.
+
+### Prerequisites for scheduled backup
+
+The prerequisites for the scheduled backup is the same with the [prerequisites for ad-hoc backup](#prerequisites-for-ad-hoc-backup).
+
+### Scheduled backup process
+
+Create the `BackupSchedule` CR to enable the scheduled full backup to GCS:
```
Comment (lilin90) [inline en/backup-to-gcs.md]: ditto for this section.

File: en/backup-to-s3.md [hunk 33]
```diff
@@ -0,0 +1,271 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic backup-demo1-tidb-secret --from-literal=password=<password> --namespace=test1
+    ```
+
+### Ad-hoc backup process
+
++ Create the `Backup` CR and back up data to Amazon S3:
```
Comment (lilin90) [inline en/backup-to-s3.md]: Process should organized in an ordered list.

**Comment by lilin90** [comment]
@TomShawn I added comments in three batches and will have another review after they are addressed.

File: en/backup-to-s3.md [hunk 34]
```diff
@@ -0,0 +1,287 @@
...
+            storageClassName: local-storage
+            storageSize: 10Gi
+        ```
+
+    2. Create the `BackupSchedule` CR to enable the scheduled full backup to Amazon S3:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl apply -f backup-schedule-s3.yaml
```
Comment (TomShawn) [inline en/backup-to-s3.md]: @onlymellb Please help confirm whether the file name `backup-schedule-s3.yaml` in the command is correct, thanks!

File: en/backup-to-s3.md [hunk 35]
```diff
@@ -0,0 +1,287 @@
...
+            storageClassName: local-storage
+            storageSize: 10Gi
+        ```
+
+    2. Create the `BackupSchedule` CR to enable the scheduled full backup to Ceph:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl apply -f backup-schedule-s3.yaml
```
Comment (TomShawn) [inline en/backup-to-s3.md]: @onlymellb Please help confirm whether the file name `backup-schedule-s3.yaml` in the command is correct, thanks!

**Comment by TomShawn** [comment]
@lilin90 Comments addressed, PTAL again, thanks!

File: en/backup-to-gcs.md [hunk 36]
```diff
@@ -0,0 +1,208 @@
+---
+title: Back up TiDB Cluster Data to GCS
+summary: Learn how to back up the TiDB cluster to GCS.
+category: how-to
+---
+
+# Back up TiDB Cluster Data to GCS
```
Comment (ran-huang) [inline en/backup-to-gcs.md]: Please take a look at https://github.com/pingcap/docs-tidb-operator/pull/19. 
Would it be better to specify the tools in the title? If so, let's make the English titles consistent with the Chinese ones.

**Pull request merged**